### PR TITLE
Prepare for new wiki build system

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

This is a first draft of the new wiki build system. It should make things a lot easier for devs. We will also provide a v2 of the wiki-cli to make things even easier ;)
I will keep this a draft until it is ready, but now if you want you can already look at what we are creating.
The wiki can be used by just running `yarn`, `yarn start` and navigating to http://localhost:3000/chrysalis-docs/welcome

TO-DOs:
- [x] Test
- [x] Add v2 of Wiki CLI

## Type of change

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] Enhancement (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Fix

### CODE REVIEW

**Comment by Dr-Electron** [comment]
Ok, most of the things should work now :trollface:
You can test the docs locally with `yarn start`
I also added a test build to check for broken links and removed the mqtt links as they aren't proper links anyway.
Also please inform the Wiki team (on discord) once you plan to merge as we need to update the wiki that day too :)

File: DOCUMENATION_README.md [hunk 1]
```diff
@@ -10,28 +10,18 @@ The documentation is built using [Docusaurus 2](https://docusaurus.io/). The dep
 ## Installation
 
 ```console
-npm i
-npm run setup
+yarn
 ```
 
-This command checks out a local copy of the wiki and creates links to the content.
+This command installs all needed dependencies.
```
Comment (charlesthompson3) [inline DOCUMENATION_README.md:16]:
```suggestion
This command installs all necessary dependencies.
```

File: DOCUMENATION_README.md [hunk 2]
```diff
@@ -10,28 +10,18 @@ The documentation is built using [Docusaurus 2](https://docusaurus.io/). The dep
...
-
-## Tear Down
-
-```console
-npm run clean
-```
-
-This command deletes the local wiki and local links.
-
+This command starts a local, wiki themed, development server and opens up a browser window. Most changes are reflected live without having to restart the server.
```
Comment (charlesthompson3) [inline DOCUMENATION_README.md:24]:
```suggestion
This command starts a local, wiki themed development server and opens up a browser window. Most changes are reflected live without having to restart the server.
```

**Review by charlesthompson3 — APPROVED** [review]
LGTM 👌

**Pull request merged**